### DISCUSSION TITLE
part-ii/2016-3/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#179] [Question comments]
Part II 2016 Algebraic Geometry: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2016-3/

**Comment by silkca** [comment]
The first part is bookwork.

The second part is relatively simple, and I believe was used on an example sheet.

For the final part, I chose to explicitly calculate in the same way as the second part, which is surprisingly not too bad. There are simpler methods by considering it as the composition of two Segre embeddings, which avoids such large ideals and produces the same result.

**Reply by eha1234** [reply]
I'm not sure if this is the same, but you can find an explicit inverse if you choose a particular ideal. 

<details><summary>Spoilers</summary>

$I = \langle x_1 x_2 - x_0 x_3, x_1 x_4 - x_0 x_5, x_2 x_4 - x_0 x_6, x_4 x_2 x_1 - x_0^2 x^7\rangle$, then consider $(x_0,...,x_7) \mapsto ((x_0:x_4), (x_0:x_2), (x_0:x_1))$
</details>
